.github/workflows/push.yml

```diff
@@ -14,6 +14,8 @@ jobs:
         with:
           architecture: x64
           version: ${{ matrix.racket-version }}
-      - run: raco pkg install --batch --auto --name rackcheck
-      - run: raco test .
-      - run: raco test examples/*
+      - run: raco pkg install --batch --auto ./rackcheck-lib
```

Comment:
You can merge this line and the next into

```
- run: raco pkg install --batch --auto rackcheck-lib/ rackcheck/
```

Reply:
Done https://github.com/Bogdanp/rackcheck/pull/10/commits/d7dfdef9403b5cf09220046e5fee95ded7aeb0a8